### DISCUSSION TITLE
Community assist for Apache pulsar opa plugin like Kafka plugin  https://github.com/StyraInc/opa-kafka-plugin

### DISCUSSION THREAD

**Discussion by muni-chada** [open-policy-agent/community#340] [Community]
Request for Community assistance for Apache pulsar opa plugin like Kafka plugin  https://github.com/StyraInc/opa-kafka-plugin

**Comment by charlieegan3** [comment]
Hi @muni-chada, this is a good question - I don't think we have a way to do this at the moment. You might get a little more visibility in the [OPA Slack](https://communityinviter.com/apps/openpolicyagent/signup). I can't promise someone will jump to implement this, but it'd be a good place to go to get support in building something like this.

**Reply by anderseknert** [reply]
Also, see https://github.com/StyraInc/opa-kafka-plugin/issues/45
